File: Examples/Node.js/Document/CreateEmptyPdf.js
// ExStart:1
var fs = require('fs');
var assert = require('assert');
var PdfApi = require('asposepdfcloud');
var StorageApi = require('asposestoragecloud');
var configProps = require('../config/config.json');
var data_path = '../../../Data/';

var AppSID = configProps.app_sid;
var AppKey = configProps.api_key;
var outFolder = configProps.out_folder;
var config = {'appSid':AppSID,'apiKey':AppKey , 'debug' : true};

// Instantiate Aspose Storage API SDK
var storageApi = new StorageApi(config);
// Instantiate Aspose.Pdf API SDK
var pdfApi = new PdfApi(config);

// Set input file name
var name = "CreateEmptyPdf.pdf";

try {
// Invoke Aspose.Pdf Cloud SDK API to create an empty PDF file
pdfApi.PutCreateDocument(name, null, null, null, null, null, function(responseMessage) {
		assert.equal(responseMessage.status, 'OK');
		console.log("Empty Pdf has been created successfully");

		// Download pdf from storage server
		storageApi.GetDownload(name, null, null, function(responseMessage) {
			assert.equal(responseMessage.status, 'OK');
			var writeStream = fs.createWriteStream(data_path + "CreateEmptyPdf_out.pdf");
			writeStream.write(responseMessage.body);
			});
	});

}catch (e) 
{
  console.log("exception in example");
  console.log(e);
}
//ExEnd:1

File: Examples/Python/Examples/CreateEmptyPDF.py
import asposepdfcloud
from asposepdfcloud.PdfApi import PdfApi
from asposepdfcloud.PdfApi import ApiException

import asposestoragecloud
from asposestoragecloud.StorageApi import StorageApi
from asposestoragecloud.StorageApi import ResponseMessage

import random
import string

apiKey = "XXXXX" #sepcify App Key
appSid = "XXXXX" #sepcify App SID
apiServer = "http://api.aspose.com/v1.1"
data_folder = "../../data/"

#Instantiate Aspose Storage API SDK
storage_apiClient = asposestoragecloud.ApiClient.ApiClient(apiKey, appSid, True)
storageApi = StorageApi(storage_apiClient)
#Instantiate Aspose Pdf API SDK
api_client = asposepdfcloud.ApiClient.ApiClient(apiKey, appSid, True)
pdfApi = PdfApi(api_client);

#set input file name
name = ''.join(random.choice(string.ascii_uppercase + string.digits) for _ in range(8))
name = name + ".pdf"

try:
    #invoke Aspose.Pdf Cloud SDK API to create an empty PDF file
    response = pdfApi.PutCreateDocument(name)

    if response.Status == "OK":
        print "Empty Pdf has been created successfully"
        #download pdf from storage server
        response = storageApi.GetDownload(Path=name)
        outfilename = "c:/temp/" + name
        with open(outfilename, 'wb') as f:
                    for chunk in response.InputStream:
                        f.write(chunk)

except ApiException as ex:
            print "ApiException:"
            print "Code:" + str(ex.code)
            print "Message:" + ex.message

